scripts/ci-conformance.sh

```diff
@@ -71,7 +71,8 @@ build_k8s() {
     # ensure the e2e script will find our binaries ...
     mkdir -p "${PWD}/_output/bin/"
     cp -f "${PWD}/bazel-bin/test/e2e/e2e.test" "${PWD}/_output/bin/e2e.test"
-    cp -f "${PWD}/bazel-bin/vendor/github.com/onsi/ginkgo/ginkgo/darwin_amd64_stripped/ginkgo" "${PWD}/_output/bin/ginkgo"
+    # workaround for mac os
+    cp -f "${PWD}/bazel-bin/vendor/github.com/onsi/ginkgo/ginkgo/darwin_amd64_stripped/ginkgo" "${PWD}/_output/bin/ginkgo" || true
```

Comment:
@detiber did you have to do this to run aws conformance locally?

Reply:
I did not, but I also run a Linux desktop.